features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/des/des_alt_sw.h

```diff
@@ -145,7 +137,7 @@ int mbedtls_des_sw_setkey_dec( mbedtls_des_sw_context *ctx, const unsigned char
  * \return         0
  */
 int mbedtls_des3_sw_set2key_enc( mbedtls_des3_sw_context *ctx,
```

Comment:
I am not sure I understand correctly how this works. There seems to be two sets of files: des_alt_sw and des_alt. Clearly, the former is a software implementation (pretty much a copy of the default mbed TLS des module) while the latter is an implementation using the hardware accelerator. However, how is it decided which implementation is used? When would the software implementaiton be used? Note that the function names are different.

Reply:
@andresag01 Yes, S/W implementation was duplicated to test H/W port but I forgot to remove it. Now it is removed in https://github.com/ARMmbed/mbed-os/pull/4925/commits/6c54139e10e96402afe2e9ec032190af8c9791a5.